README.md

```diff
@@ -1,30 +1,24 @@
-# Pants Build System
+# Pantsbuild
 
-Pants is a scalable build system for _monorepos_: codebases containing 
-multiple projects, often using multiple programming languages and frameworks, 
-in a single unified code repository.
+Pantsbuild is a fast, scalable, user-friendly build system for codebases of 
+all sizes. It's currently focused on Python, Shell, Docker, and Go, with support for 
```

Comment:
I'd worry about saying "focus" as our support becomes less focused on a few languages.

Maybe "It currently has support for"?